docs/ops/matrix/Einsum_7.md

```diff
@@ -0,0 +1,248 @@
+the second and third dimensions. In case ellipsis presents in input subscripts for several operands, the dimensions covered by the ellipsis
+must be broadcastable to satisfy numpy broadcasting (or multidirectional broadcasting) rules available in
+<a href="https://numpy.org/doc/stable/user/basics.broadcasting.html#general-broadcasting-rules">numpy broadcasting documentation</a>.
+If ellipsis presents in at least one input subscript, the output subscript must always contain one ellipsis.
+For example, *Einsum* operation on two inputs (of shapes `[11, 1, 4, 3]` and `[3, 11, 7, 1]`) with `equation="a...b,b...->a..."`
+has ellipsis for both operands covering dimensions with sizes `[1, 4]` and `[11, 7, 1]` that are broadcasted to `[11, 7, 4]`.
+The resulted shape of *Einsum* operation will be `[11, 11, 7, 4]` since a dimension labeled with `a` is left alongside with broadcasted dimensions.
+
+Example 6 shows how *Einsum* operates on the single input with an equation containing ellipsis:
+
+```
+A = [[1.0, 2.0, 3.0],
+      [4.0, 5.0, 6.0],
+      [7.0, 8.0, 9.0]]
+equation = "a...->..."
+output = [12.0, 15.0, 17.0])
+```
+
+Example 7 shows how *Einsum* operates with broadcasting two operands:
+
+```
+A = [[1.0, 2.0, 3.0],
+      [4.0, 5.0, 6.0],
+      [7.0, 8.0, 9.0]]
+B = [0.5]
+equation = "a...,...->a..."
+output = [[0.5, 1.0, 1.5],
+          [2.0, 2.5, 3.0],
+          [3.5, 4.0, 4.5]]
+```
+
+**NOTE**: Deep learning frameworks support implicit mode (a classical form of Einstein summation) where the equation does not have the output subscript 
+and has the following format: `<subscript for input1>, <subscript for input2>, ..., <subscript for inputn>`.
+The equation consists of only input subscripts for each operand.
+The output subscript can be recovered as a sequence of alphabetically sorted labels that do not repeat in the left-hand of the equation.
+For example, `equation = "dbbc,ca"` in implicit mode is the same as `equation = "dbbc,ca->ad"`.
+The equation in implicit mode can set up only subset of Einstein summation conventions so this mode has limitations. For example,
+`equation = "kii->i"` cannot be represented in implicit mode. *Einsum* operation only supports explicit mode. In case of ellipsis label in the left-hand
+of the equation in implicit mode, the ellipsis will come first in the output subscript for the recovery.
+
+The *Einsum* operation has *einsum_path* attribute that can recommend to plugins an order of input contraction. In case of multiple inputs
+the effective order can boost performance and reduce memory costs during *Einsum* computation. The order of applying of *Einsum* operation to inputs 
+makes sense in terms of performance. For example, *Einsum* operation on three inputs (of shapes `[2, 2]`, `[2, 5]` and `[5, 2]`) 
+with `equation="ij,jk,kl->il"` will behave more effective if the operation performs on the second and the third inputs first to avoid 
```

Comment:
performs -> is performed

Reply:
"performs" is correct here. I try to avoid "passive voice" where it is possible. Anyway, I asked our tech-writers and they would check it.